watchtower/wtclient/client.go

```diff
@@ -293,19 +295,6 @@ func New(config *Config) (*TowerClient, error) {
 	// new sessions will be requested.
```

Comment:
the comment seems to be outdated

Reply:
is it?

Reply:
I think you don't load the sessions anymore? I haven't looked into the details, though - you're the expert in this matter :)

Reply:
oh! I see - sorry, I was looking at this comment on the wrong commit. Cool yeah the comment is just out of place. Have moved it to where it makes more sense. Good catch!